File: year2020/month10/day1023/ms_1023.cpp
#include <iostream>
#include <cstring>

using namespace std;

int main() {
	string first;
	string second;

	cin >> first;
	cin >> second;

	if (first.length() < 20 && second.length() < 20)
		cout << first.length() + second.length() << endl;

	return 0;
}

File: year2020/month10/day1023/ms_1023.cs
using System;

namespace algorithm_95
{
    class ms_1023
    {
        static void Main(string[] args)
        {
            String first = Console.ReadLine();
            String second = Console.ReadLine();

            if(first.Length < 20 && second.Length < 20)
                Console.WriteLine(first.Length + second.Length);
        }
    }
}
